extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -14,83 +14,80 @@
 public class OidcCommonConfig {
     /**
      * The base URL of the OpenID Connect (OIDC) server, for example, `https://host:port/auth`.
-     * OIDC discovery endpoint will be called by default by appending a '.well-known/openid-configuration' path to this URL.
-     * Note if you work with Keycloak OIDC server, make sure the base URL is in the following format:
-     * `https://host:port/realms/{realm}` where `{realm}` has to be replaced by the name of the Keycloak realm.
+     * Set this property only if the public key verification {@link #publicKey} or certificate chain verification only
```

Comment:
Sorry, just one more suggestion:

```suggestion
     * Do not set this property if the public key verification {@link #publicKey} or certificate chain verification only are required.
```

and make it the last line in this javaDocs fragment.

Reply:
Updated. Thanks.